src/utils/hooks/useScript.ts

```diff
@@ -0,0 +1,23 @@
+import React, {useEffect} from "react";
+
+/*  
+Custom hook to append the Google Identity Services script to the Index.html file without directly saving changes to the index file.
```

Comment:
why do we need this?

Reply:
if our code is deprecated -- do we need to delete the deprecated code?

Reply:
Google wants you to put a script into the main html file, so I wrote this to achieve similar results without making any changes to that file directly.

And yes, a lot of the old code pertaining to it can be deleted.

Reply:
so youre saying no backend handling is needed on our end ?

Reply:
If I'm understanding correctly, yes. The only backend handling is taking user info response from Google and creating a user in the db with it.